pkgs/os-specific/linux/usbguard/default.nix

```diff
@@ -1,28 +1,40 @@
-{
-  stdenv, fetchurl, lib,
-  pkgconfig, libxslt, libxml2, docbook_xml_dtd_45, docbook_xsl, asciidoc,
-  dbus-glib, libcap_ng, libqb, libseccomp, polkit, protobuf,
-  audit,
-  libgcrypt ? null,
-  libsodium ? null
+{ stdenv
+, lib
+, fetchFromGitHub
+, autoreconfHook
+, asciidoc
+, pkgconfig
+, libxslt
+, libxml2
+, docbook_xml_dtd_45
+, docbook_xsl
+, dbus-glib
+, libcap_ng
+, libqb
+, libseccomp
+, polkit
+, protobuf
+, audit
+, libgcrypt ? null
+, libsodium ? null
```

Comment:
```suggestion
, libgcrypt
, libsodium
```

Reply:
Out of interest what's the reason for this change? `? null` seems pretty common for optional dependencies across nixpkgs

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/misc/polybar/default.nix#L6-L14

Reply:
It is not required if you use optional dependencies with lib.optional(s). 

The package you linked is a bit outdated and wouldn't get a merge from me if anything near those lines changes. Especially this bit https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/misc/polybar/default.nix#L60 is scary.

Reply:
Ah ok, awesome. Informative as always! :)